detection_rules/main.py

```diff
@@ -415,8 +417,16 @@ def test_rules(ctx):
     """Run unit tests over all of the rules."""
     import pytest
 
+    rules_config = ctx.obj['rules_config']
+    test_config = rules_config.test_config
+    tests, skipped = test_config.get_test_names(formatted=True)
+
+    if skipped:
+        click.echo(f'Tests skipped per config ({len(skipped)}):')
+        click.echo('\n'.join(skipped))
+
     clear_caches()
-    ctx.exit(pytest.main(["-v"]))
+    ctx.exit(pytest.main(['-v'] + tests))
```

Comment:
I initially built a custom class around `unittest.TestLoader`, but since we build tests in unittest and call via pytest (for formatting), it was moot and easier to just pass as a command line arg